deploy/charts/rook-ceph-cluster/prometheus/localrules.yaml

```diff
@@ -837,25 +824,7 @@ groups:
             orchestrator from service management and reconciliation. If this is
             not intentional, resume cephadm operations with 'ceph orch resume'
```

Comment:
Wouldn't these alerts also be non-applicable to Rook?

Reply:
Good catch, I'll remove the cephadm-specific alerts as well even though they aren't firing.